tests/test_docs.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+import subprocess
+import re
+import logging
+import pytest
+import os
+
+if not os.getenv("CI") or os.getenv("SKIP_DOCS_TEST", False):
+    pytest.skip("Skipping docs test", allow_module_level=True)
```

Comment:
I don't know if this is necessary. Nobody will be able to run `pytest` without specifying which tests to run, since we have a few tests that depend on hardware. But if we do go with this, we'd need to pair it with setting the appropriate environment variable in `test_pr.yml` or it won't run.